source/DistanceAndDirection/ProAppDistanceAndDirectionModule/ProAppDistanceAndDirectionModule/ViewModels/ProCircleViewModel.cs

```diff
@@ -539,14 +540,29 @@ public override string DistanceString
             {
                 if (CircleType == CircleFromTypes.Diameter)
                 {
-                    return (Distance * 2.0).ToString("G");
+                    if (EndsWithDecimal)
+                        return (Distance * 2.0).ToString("G") + ".";
+                    else
+                        return (Distance * 2.0).ToString("G");
                 }
-
-                return base.DistanceString;
+                if (EndsWithDecimal)
+                    return base.DistanceString + ".";
+                else
+                    return base.DistanceString;
             }
             set
             {
-                // lets avoid an infinite loop here
+                //Handle for decimals
+                if(value.EndsWith("."))
```

Comment:
Just checking if there are there any i18n issues with using this constant (ex. "." vs. ",")